samples/bluetooth/hci_rpmsg/src/main.c

```diff
@@ -234,7 +234,8 @@ static int hci_rpmsg_send(struct net_buf *buf)
 #if defined(CONFIG_BT_CTLR_ASSERT_HANDLER)
 void bt_ctlr_assert_handle(char *file, uint32_t line)
 {
-	LOG_ERR("Controller assert in: %s at %d", file, line);
+	LOG_ERR("Controller assert in: %s at %d", log_strdup(file), line);
```

Comment:
Wouldn't it be better to use `ASSERT()` here?

Reply:
BT_ASSERT(false); you mean?

Reply:
Do you agree if I use BT_ASSERT_MSG() to replace these two lines?

Reply:
Updated: Using BT_ASSERT_MSG and disabling debug message as before.